packages/ui/src/theme/tokens/components/expander.ts

```diff
@@ -1,17 +1,96 @@
-import { DesignToken } from '../types/designToken';
+import {
+  AlignItemsValue,
+  AnimationDurationValue,
+  AnimationTimingFunctionValue,
+  BorderRadiusValue,
+  BoxShadowValue,
+  ColorValue,
+  DesignToken,
+  DisplayValue,
+  JustifyContentValue,
+  RadiusValue,
+  SpaceValue,
+  TransitionDurationValue,
+  TransitionTimingFunctionValue,
+} from '../types/designToken';
+
+interface ExpanderItemFocusTokens {
+  boxShadow: DesignToken<BoxShadowValue>;
+}
+
+interface ExpanderItemTokens {
+  marginTop: DesignToken<SpaceValue>;
+  boxShadow: DesignToken<BoxShadowValue>;
+  borderBottomLeftRadius: DesignToken<RadiusValue>;
+  borderBottomRightRadius: DesignToken<RadiusValue>;
+  borderTopLeftRadius: DesignToken<RadiusValue>;
```

Comment:
Why do we have both positional (top/left) and semantic (start/end) versions of border radius? Should we stick with one?

Reply:
That's very strange indeed, I'll create a ticket for this.

Reply:
Oh now I remember, it's because Safari didn't support semantic border radius, so we had to have both specified in the CSS. But I don't think there's a reason to keep both tokens, we can just use the same positional tokens.